README.md

```diff
@@ -1 +1,70 @@
-# mod1-aq-hello
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Module 1 Day 2 Hello World Repo
+
+> In this project, you will set up a "Hello world" repository. No complex coding is required for this exercise. Your goal here is to master all of the tools and best practices you learned about in previous steps. 
+
+
+![screenshot](./task.png)
+
+You will be using them in all Microverse projects and most likely in your future job as well, so it is important to know them!
+
+## Built With
+
+- HTML - CSS
+- Git - Github - GitFlow
+
+## Live Demo
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
+
+## Authors
+
+👤 **Author : Ramiro Aquistapace**
+
+- GitHub: [@ramiaquista](https://github.com/ramiaquista)
+- Twitter: [@ramiaq1](https://twitter.com/ramiaq1)
+- LinkedIn: [RamiroAquistapace](https://www.linkedin.com/in/ramiro-aquistapace-32b61b204/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] You should update your README file. Make sure there are no  broken links(example screenshot, license) and empty sections. If you won't use some sections, delete them. 
- [ ] I recommend using as many sections as you can  
- [ ] You should change the template's sentence with your sentence.